packages/utils-general/src/Countries.ts

```diff
@@ -0,0 +1,1496 @@
+export const Countries = [
```

Comment:
Do we really want to store this in the repo instead of using a library like https://github.com/michaelwittig/node-i18n-iso-countries?
Is this extracted from the I-REC api?

Reply:
Yes, it is extracted from I-REC API.